tests/test_sap_success_factors/test_learner_data.py

```diff
@@ -280,25 +334,98 @@ def get_course_grade(course_id, username):
             )
         mock_grades_api.return_value.get_course_grade.side_effect = get_course_grade
 
+        # Mock enrollment data
+        mock_enrollment_api.return_value.get_course_enrollment.return_value = dict(
+            mode="verified"
+        )
+
         # Collect the learner data, with time set to NOW
         with freeze_time(self.NOW):
             learner_data = list(self.exporter.collect_learner_data())
 
         assert len(learner_data) == 3
-        assert learner_data[0].enterprise_course_enrollment_id == enrollment1.id
-        assert learner_data[0].course_id == self.course_id
-        assert not learner_data[0].course_completed
-        assert learner_data[0].completed_timestamp is None
-        assert learner_data[0].grade == BaseLearnerExporter.GRADE_INCOMPLETE
-
-        assert learner_data[1].enterprise_course_enrollment_id == enrollment3.id
-        assert learner_data[1].course_id == self.course_id
-        assert not learner_data[1].course_completed
-        assert learner_data[1].completed_timestamp is None
-        assert learner_data[1].grade == BaseLearnerExporter.GRADE_INCOMPLETE
-
-        assert learner_data[2].enterprise_course_enrollment_id == enrollment2.id
-        assert learner_data[2].course_id == course_id2
-        assert learner_data[2].course_completed
-        assert learner_data[2].completed_timestamp == self.NOW_TIMESTAMP
-        assert learner_data[2].grade == grade
+
+        report1 = learner_data[0]
```

Comment:
Not really, that's why we were just referring to them as `learner_data[x]` before. Notice above in the code we have `enrollment1`, `enrollment2`, etc..